client/core/core.go

```diff
@@ -4740,10 +4740,10 @@ func (c *Core) Login(pw []byte) error {
 		// and the balance updated there.
 		c.notify(newLoginNote("Connecting wallets..."))
 		c.connectWallets() // initialize reserves
-		c.notify(newLoginNote("Resuming active trades..."))
-		c.resolveActiveTrades(crypter)
 		c.notify(newLoginNote("Connecting to DEX servers..."))
 		c.initializeDEXConnections(crypter)
+		c.notify(newLoginNote("Resuming active trades..."))
+		c.resolveActiveTrades(crypter)
```

Comment:
Seems dangerous. What if the DEX server sends info about an active trade that we haven't loaded from the database yet?

Reply:
Hmm, then a better resolution to this would be to check if we are logged in? https://github.com/decred/dcrdex/issues/2617#issuecomment-1859427427, but we can just log a msg and not exit.

Reply:
Yeah maybe.